azure-pipelines.yml

```diff
@@ -100,6 +98,7 @@ jobs:
     - task: NuGetToolInstaller@0
```

Comment:
A few lines above this, I see we're doing a `Release` build configuration. Let's actually switch this back to `Debug` since this is not an official release package. That will make testing easier.